src/erlcloud_ecs.erl

```diff
@@ -163,7 +163,10 @@
     volume_name_opt/0,
     volumes/0,
     volumes_opt/0,
-    volumes_opts/0
+    volumes_opts/0,
+    placement_strategy/0,
```

Comment:
It looks like these are exported alphabetically. Any reason not to put them in accordingly?

Reply:
sure